extensions/azdata/src/typings/azdata.d.ts

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
do we think it may be confusing to add a new file named `azdata.d.ts` that contains a different API definition than we use in the other `azdata.d.ts` file?

Reply:
Updated to azdata-ext